src/components/feedback-message/README.md

```diff
@@ -0,0 +1,31 @@
+# Feedback Message
+
+A feedback message component to provide feedback to the user.
+
+## Usage
+
+```jsx
+import { FeedbackMessage } from '@idm/web-uikit';
+
+<FeedbackMessage>Generic feedback</FeedbackMessage>
+
+// Large variant
+<FeedbackMessage variant="large">Generic feedback: large variant</FeedbackMessage>
+
+// Feedback type error
+<FeedbackMessage type="error">Something went wrong!</FeedbackMessage>
+```
+
+## Props
+| name            | type   | default              | description                                                                                                     |
+|-----------------|--------|----------------------|-----------------------------------------------------------------------------------------------------------------|
```

Comment:
Nift pick: can we avoid having huge table definitions like `----------------`? It's not necessary at all for them to align. I would just put enough `-` chars. If you agree, please change it in every file. For instance, the `Icon` readme is good!

Reply:
Personally, I prefer having the markdown source well formatted. I feel it makes it more readable and avoids having to count table cells to know if you're modifying the correct column.